projects/go-lib/src/lib/components/go-copy/go-copy.component.scss

```diff
@@ -1,10 +1,14 @@
+.go-copy {
```

Comment:
since this is being added to the host do we have to specify this as a host attribute?
```
:host.go-copy {
  position: relative;
}
```

Reply:
We don't! This is what the `@HostBinding()` allows us to do. We can keep the scoped styles in the template without having to use `:host`.

Reply:
@grahamhency looks like while go-copy is currently being added to the host property, the css applied here is not being applied to the host property in the deploy preview.

Reply:
Well shit, you're right. I remember not needing to add that at some point... perhaps I remembered wrong. :shrug: I added the `:host` portion in the SCSS.